### ISSUE TITLE
Failure with windows images

### ISSUE THREAD

**Issue by briandealwis** [GoogleContainerTools/container-structure-test#197]
In trying to [upgrade bazelbuild/rules_docker to use container-structure-test](https://github.com/bazelbuild/rules_docker/pull/685), we hit an odd test failure with performing tests against windows images:
```
$ bazel-out/darwin-fastbuild/bin/structure_test_at_workspace_root.runfiles/structure_test_darwin/file/downloaded version
v1.7.0
$ bazel-out/darwin-fastbuild/bin/structure_test_at_workspace_root.runfiles/structure_test_darwin/file/downloaded test --driver tar --image bazel-out/darwin-fastbuild/bin/tests/docker/basic_windows_image.tar --config tests/docker/configs/windows_image.yaml

===========================================
====== Test file: windows_image.yaml ======
===========================================
=== RUN: Files
--- FAIL
Error: error creating driver: processing tar image reference: retrieving tar from path: EOF

=== RUN: Hives
--- FAIL
Error: error creating driver: processing tar image reference: retrieving tar from path: EOF

=== RUN: Metadata Test
--- FAIL
Error: error creating driver: processing tar image reference: retrieving tar from path: EOF


===========================================
================= RESULTS =================
===========================================
Passes:      0
Failures:    3
Total tests: 3

FAIL
FATA[0000] FAIL
```

Steps to repeat:
  - `git clone https://github.com/bazelbuild/rules_docker`
  - `cd rules_docker`
  - `git pull origin pull/685/head` (pulls the branch to update to 1.7.0)
  - `bazel test "--define=ENV_KEY=my_key" "--define=ENV_VALUE=my_value" //tests/docker:basic_windows_image_test`

**Comment by briandealwis** [comment]
I believe the problem is that the rules_docker tests are somehow creating images with 0-byte `layer.tar` files.  Details on 
https://github.com/bazelbuild/rules_docker/pull/685#issuecomment-461179681

**Comment by briandealwis** [comment]
Bug was in google/go-containerregistry#367 and now fixed google/go-containerregistry#368

**Comment by nkubala** [comment]
I just released v1.8.0 which should have the fix from go-containerregistry picked up. @briandealwis can you verify it's fixed and close this issue?

**Comment by briandealwis** [comment]
Works for me!  Thanks @nkubala!
